### ISSUE TITLE
Error while resolving settings from workspace - directs to log that does not include the error

### ISSUE THREAD

**Issue by chazmo03** [astral-sh/ruff-vscode#664]
In the case where the ruff binary does not meet the required version from the configuration (in `pyproject.toml`, for example), the below error warning is raised. The **Show Logs** button directs to the output panel for Ruff, but those logs do not show any errors or warnings. In my case, the error is due to the ruff binary being incompatible with the `required-version` in the configuration.

It would be helpful if this error was surfaced, either in the notification or in the logs.

![Image](https://github.com/user-attachments/assets/a9032303-47ee-4700-b835-0aae0e0e1357)

If I run the ruff binary from the terminal, it fails with the following:

```text
❯ ruff format .
ruff failed
  Cause: Required version `>=0.9.0` does not match the running version `0.8.5`
```

**Comment by MichaReiser** [comment]
Oh, that's interesting. Are you using the latest extension (2025) or only the latest ruff version? We made some changes around this in the most recent extension

**Comment by dhruvmanila** [comment]
We fixed the observability issues like this one in the latest version. The "Show Logs" button should now open the server logs which should contain them:

```
   0.008362166s ERROR ThreadId(04) ruff_server::session::index::ruff_settings: Failed to resolve settings for /Users/dhruv/playground/ruff/pyproject.toml: Required version `>=0.9.0` does not match the running version `0.8.4`
   0.016165833s  INFO main ruff_server::session::index: Registering workspace: /Users/dhruv/playground/ruff
   0.016933875s  INFO ruff:main ruff_server::server: Configuration file watcher successfully registered
```

**Comment by chazmo03** [comment]
What timing. I just got the notification to update to the latest (2025) version. After upgrading, the Show Logs button now takes me to the Ruff Language Server output instead of the Ruff output. But the output is empty.

**Comment by dhruvmanila** [comment]
> What timing. I just got the notification to update to the latest (2025) version. After upgrading, the Show Logs button now takes me to the Ruff Language Server output instead of the Ruff output. But the output is empty.

Sorry, you'll also need to update the Ruff version to be at least 0.9.0 as this fix required updating both Ruff and the VS Code extension.
